### ISSUE TITLE
[Bug Report] Linux epic games error

### ISSUE THREAD

**Issue by xTruesX** [ow-mods/ow-mod-man#790]
### What Happened?

I was trying to install some mods and open the game but it wont open unless i check the force exe button on the settings and that gets rid of the dlc. I want to play the game modded and with dlc. I bought the game from epic games and can open it without modded using heroic games launcher for arch linux. I tried bunch of things but it gives me the same warning every time: Cant read the file com.epicgames.launcher://apps/starfish%3A601d0668cef146bd8eef75d43c6bbb0b%3AStarfish?action=launch&silent=true is there a way around this without using the force exe button?

### Package

Graphical Interface

### Platform

Linux

### Logs

[Manager][LogServer][Info] Ready to receive game logs on port 46841!
[Manager][LogServer][Info] ====== Client Connected To Console ======
[OWML][App][Info] Started OWML v2.14.0
[OWML][App][Message] Game found in /home/omerf/Games/Heroic/OuterWilds
[OWML][GameVersionHandler][Info] Game version: 1.1.15.1018
[OWML][App][Message] Game files copied.
[OWML][App][Message] Found mods:
[OWML][App][Message] * xen.Decloaked v1.0.3 
[OWML][App][Message] * xen.EnableMeditation v1.0.0 
[OWML][App][Message] * clubby789.OWClock v0.8.1 
[OWML][App][Message] * xen.CommonCameraUtility v1.4.0 
[OWML][App][Message] * PacificEngine.CheatsMod v0.7.6 
[OWML][App][Message] * xen.BetterModelShip v0.2.1 
[OWML][OWPatcher][Message] Copying OWML files...
[OWML][OWPatcher][Message] Copying replacement OW files...
[OWML][OWPatcher][Message] Patching OW assembly...
[OWML][OWPatcher][Message] - Assembly is already patched.
[OWML][OWPatcher][Message] Updating incremental GC...
[OWML][App][Message] Identified as an Epic install. Launching...
[OWML][App][Error] Error while starting game: System.ComponentModel.Win32Exception (0x80004005): Cannot find the specified file
  at System.Diagnostics.Process.StartWithShellExecuteEx (System.Diagnostics.ProcessStartInfo startInfo) [0x00102] in <aa8bc4cb919a4978832b583874d62dd6>:0 
  at System.Diagnostics.Process.Start () [0x00032] in <aa8bc4cb919a4978832b583874d62dd6>:0 
  at (wrapper remoting-invoke-with-check) System.Diagnostics.Process.Start()
  at System.Diagnostics.Process.Start (System.Diagnostics.ProcessStartInfo startInfo) [0x0001b] in <aa8bc4cb919a4978832b583874d62dd6>:0 
  at System.Diagnostics.Process.Start (System.String fileName) [0x00006] in <aa8bc4cb919a4978832b583874d62dd6>:0 
  at OWML.Abstractions.ProcessHelper.Start (System.String path, System.String[] args) [0x0000a] in <92b1a076683a46f19672de9738d71e36>:0 
  at OWML.Launcher.App.StartGame () [0x00130] in <43156c17432e404bbd6d1bc87ac60bb5>:0 
[Manager][LogServer][Info] ====== Client Disconnected From Console ======

**Comment by Bwc9876** [comment]
Hello, It seems like the game is able to be patched to run modded. When you get the error about not being able to read the file, try launching the game manually through the Heroic launcher instead. The game should still be patched.

**Comment by xTruesX** [comment]
Yes the game looks like patched but it shows no mods installed i just have a tab called "mods" and its empty. how can i fix this?

**Comment by Bwc9876** [comment]
Did you install the manager via flatpak as well as the heroic games launcher?

**Comment by xTruesX** [comment]
Heroic is installed via flatpak but i think installed the manager from Arch repositories

**Comment by Bwc9876** [comment]
You may need to follow [the Steam flatpak part of the FAQ](https://github.com/ow-mods/ow-mod-man/blob/main/owmods_gui/HELP.md#mod-folder-not-found-on-flatpak-version-of-steam), as a similar thing happens due to access issues. Replacing the app id for Steam with the app id for heroic wherever applicable.
